core/services/feeds/service.go

```diff
@@ -1550,6 +1553,32 @@ func (s *service) isRevokable(propStatus JobProposalStatus, specStatus SpecStatu
 	return propStatus != JobProposalStatusDeleted && (specStatus == SpecStatusPending || specStatus == SpecStatusCancelled)
 }
 
+func (s *service) periodicallySyncNodeInfo(ctx context.Context) {
+	s.lggr.Info("starting periodic sync node info goroutine")
+
+	timer := time.NewTicker(s.gCfg.FeedsManagerSyncInterval())
+	for {
+		select {
+		case <-timer.C:
+			managers, err := s.ListManagers(ctx)
+			if err != nil {
+				s.lggr.Errorw("failed to list managers", "err", err)
+			}
+
+			for _, manager := range managers {
+				s.lggr.Infow("synchronizing node info", "managerID", manager.ID)
+				err := s.SyncNodeInfo(ctx, manager.ID)
```

Comment:
managers/JDs can be disabled/enabled, we should avoid synching to disabled JDs, we can use the `DisabledAt` to determine it like at line 1130 of this file